.clang-format

```diff
@@ -0,0 +1,75 @@
+---
+Language:        Cpp
+AccessModifierOffset: -2
```

Comment:
Given that our TabWidth is 4 this setting causes access modifiers to be indented using spaces (as there is no such thing as a half tab).
The other option we'd have would be to not indent them in a special way (which I don't like a lot) or to fully outdent them by 4 columns causing them to use one tab less of indentation. This would probably lead to most specifiers not being indented at all like so:
```
class Test {
public:
    int tester;
};
```
I'm not sure if this might be the better idea than setting this option to -2

Reply:
Aren't we currently indenting access modifiers with a single tab?

Reply:
Manually that's what I do, yes. But it seems that clang-format doesn't support indenting the modifiers by a full tab and then the following code by 2 tabs. The indent of the modifiers doesn't influence the indent of the following code.

That's why I chose to outdent the modifiers instead

Reply:
No indentation for the access modifiers is fine for me.

Reply:
Would you also be fine with outdenting them by a full tab (aka: don't indent the modifiers and only indent the class body)? I would like to not mix space and tab indentation wherever possible...

Reply:
Could you write an example snippet, please?

Reply:
The example snippet is the one in my first comment :D

Reply:
Oh, I see.

In that case I definitely prefer the outdent of 4.